includes/wizards/class-reader-revenue-wizard.php

```diff
@@ -402,7 +398,24 @@ public function api_update_stripe_settings( $request ) {
 		// @todo when is the best time to do this?
 		$wc_configuration_manager->set_smart_defaults();
 
-		return \rest_ensure_response( $this->fetch_all_data() );
+		return $this->fetch_all_data();
+	}
+
+	/**
+	 * Save Stripe settings.
+	 *
+	 * @param WP_REST_Request $request Request object.
+	 * @return WP_REST_Response Response.
+	 */
+	public function api_update_stripe_settings( $request ) {
+		$params = $request->get_params();
+		$result = $this->update_stripe_settings( $params );
+
+		if ( is_wp_error( $result ) ) {
+			return rest_ensure_response( $result );
```

Comment:
Since we're still just returning the result in the case of a WP_Error, why even include this condition? Also, why the different namespacing on this line vs. line 418?

Reply:
That's right, removed this bit in 7157c007b690e225a3363810f76547a5a1e053b7